src/Meshes.jl

```diff
@@ -302,23 +302,38 @@ export
 
   # intersection types
   IntersectionType,
+  
+  # no intersection
+  NoIntersection,
+  # segment-segment intersection
```

Comment:
This change should not be here

Reply:
I tried to undo all the changes to start from the original master file. 
Not sure if it worked.